flutter/test/native_sdk_integration_test.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'mocks.dart';
+import 'sentry_flutter_options_test.dart';
+
+void main() {
+  group('$NativeSdkIntegration', () {
+    setUp(() {
+      TestWidgetsFlutterBinding.ensureInitialized();
+    });
+
+    test('test default values', () async {
+      final channel = const MethodChannel('initNativeSdk');
+
+      String? methodName;
+      dynamic arguments;
+      channel.setMockMethodCallHandler((call) async {
+        methodName = call.method;
+        arguments = call.arguments;
+      });
```

Comment:
this part looks duplicated across tests, probably better if extracted to a method